transport/tchannel/config.go

```diff
@@ -26,29 +26,29 @@ import (
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/peer/hostport"
 	"go.uber.org/yarpc/x/config"
-
-	"github.com/opentracing/opentracing-go"
 )
 
 const transportName = "tchannel"
 
 // TransportConfig configures a shared TChannel transport. This is shared
 // between all TChannel outbounds and inbounds of a Dispatcher.
 //
-// 	transports:
+// TransportConfig does not have any parameters at this time.
+type TransportConfig struct{}
+
+// InboundConfig configures a TChannel inbound.
+//
+// 	inbounds:
 // 	  tchannel:
 // 	    address: :4040
-type TransportConfig struct {
+//
+// At most one TChannel inbound may be defined in a single YARPC service.
```

Comment:
I think this is a good tradeoff, and we can always add support for multiple TChannel inbounds later without a breaking release (although I doubt we'll need to).

Would like @yarpc/golang's take here too.